cpp/modmesh/python/common.cpp

```diff
@@ -100,6 +99,21 @@ void Interpreter::setup_path()
         sys.path.insert(0, path)
 _set_modmesh_path())"""";
     pybind11::exec(cmd);
+    return *this;
+}
+
+Interpreter & Interpreter::setup_process(int argc, char ** argv_in)
+{
+    std::vector<std::string> argv;
```

Comment:
This is a prototype implementation.  Better organization should be done in the future.